framework/src/actions/MaterialOutputAction.C

```diff
@@ -176,6 +176,9 @@ MaterialOutputAction::act()
         }
       }
     }
+    else if (output_properties.size() && (outputs.find("none") != outputs.end()))
```

Comment:
(outputs.find("none") != outputs.end()) condition shouldn't be needed. It will always be the case if it reaches this part of the else if statement

Reply:
good call
updated

Reply:
Alright, good to go from me